src/main/java/io/odpf/firehose/sink/http/SerializableHttpResponse.java

```diff
@@ -0,0 +1,36 @@
+package io.odpf.firehose.sink.http;
+
+import org.apache.http.HttpResponse;
+import org.apache.logging.log4j.util.Strings;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Serializable;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SerializableHttpResponse implements Serializable {
```

Comment:
what is the purpose of this class? 
we can just call the debug method, if debug level is enabled.